src/screens/Lending/DepositTransactionConfirm.js

```diff
@@ -80,7 +76,7 @@ const DepositTransactionConfirm = ({
       accountAddress,
       depositAmount,
       depositAsset,
-      txFeeInfo?.fee || new BigNumber(0),
+      feeInfo?.fee || new BigNumber(0),
```

Comment:
Is this why we have the 0 gas estimation? Just wondering... is it worth displaying something other than 0? I'm just thinking that this is one of the flash points that are reported - 0 gas fee estimate - would a message be better here or is it possible that actually - something might cost 0 gas?

Reply:
Thanks, it was some fallback value leftover, completely removed 👍